Blish HUD/GameServices/ArcDpsService.cs

```diff
@@ -11,11 +11,17 @@
 namespace Blish_HUD {
 
     public class ArcDpsService : GameService {
+        private static readonly Logger Logger = Logger.GetLogger<ArcDpsService>();
 
         private static readonly object WatchLock = new object();
         #if DEBUG
         public static long Counter;
-        #endif
+#endif
```

Comment:
the if and endif are not matching in indentation

Reply:
Fixed